gui-apps/grimblast/grimblast-9999.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="A helper for screenshots within hyprland"
+HOMEPAGE="https://github.com/hyprwm/contrib/"
+
+if [[ ${PV} = 9999* ]]; then
+	EGIT_REPO_URI="https://github.com/hyprwm/contrib.git/"
+	inherit git-r3
+	S="${WORKDIR}/grimblast-9999/${PN}"
+else
+	SRC_URI="https://github.com/hyprwm/contrib/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+	S="${WORKDIR}/contrib-${COMMIT}/${PN}"
+fi
+
+LICENSE="MIT"
+SLOT="0"
+
+RDEPEND="
+	app-misc/jq
+	gui-apps/grim
+	gui-apps/slurp
+	gui-apps/wl-clipboard
+	gui-wm/hyprland
+	x11-libs/libnotify
+"
+
+DEPEND="${RDEPEND}"
+
+BDEPEND=""
+
+src_install() {
+	dodoc grimblast.1
```

Comment:
This can possibly be improved to something like
```
src_install() {
  emake PREFIX="${ED}/usr" -C grimblast
  emake PREFIX="${ED}/usr" -C shellevents
}
```

Adjust the above as needed. This way it installs all available packages inside the contrib repository.

Reply:
I updated the ebuild to allow installing shellevents too.
I also introduced two USE flags to choose which one(s) you want to install. In my case I don't want shellevents so I can just unset the USE flag and keep only grimblast.

Reply:
sounds good.